pages/common/docker.md

```diff
@@ -18,3 +18,11 @@
 - Stop a container
 
 `docker stop {{container}}`
+
+- List all images
```

Comment:
I agree, but I guess I'm unclear if this utility is supposed outline the most used commands or provide helpful tips. I used previous commands here as reference, like `docker ps`, which is also in the command's help.

I just took it off to keep the list shorter